app-template/config-template.xml

```diff
@@ -51,14 +47,17 @@
     <plugin name="cordova-plugin-android-fingerprint-auth" spec="1.4.3" />
     <plugin name="cordova-plugin-device" spec="2.0.1" />
     <plugin name="cordova-plugin-globalization" spec="1.0.9" />
-    <plugin name="cordova-plugin-qrscanner" spec="2.5.0" />
+    <plugin name="cordova-plugin-qrscanner" spec="2.6.0" />
+    <plugin name="cordova-plugin-add-swift-support" spec="1.7.2" />
     <plugin name="cordova-plugin-splashscreen" spec="5.0.2" />
     <plugin name="cordova-plugin-statusbar" spec="2.4.2" />
-    <plugin name="cordova-plugin-inappbrowser" spec="2.0.2" />
-    <plugin name="cordova-plugin-touch-id" spec="3.3.1" />
+    <plugin name="cordova-plugin-inappbrowser" spec="3.0.0" />
+    <plugin name="cordova-plugin-touch-id" spec="3.4.0">
+      <variable name="FACEID_USAGE_DESCRIPTION" value=" " />
```

Comment:
should we add an id here?

Reply:
No, the value is " " (empty). Description for FaceId is optional for `NSFaceIDUsageDescription` in plist file.

Reply:
ack